php/example_code/workdocs/CreateFolder.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * For more information about creating a WorkDocs application see the WorkDocs Developer Guide at
+ * https://docs.aws.amazon.com/workdocs/latest/developerguide/wd-auth-user.html
+ *
+ *
+ */
+// snippet-start:[workdocs.php.create_folder.complete]
+// snippet-start:[workdocs.php.create_folder.import]
+
+require 'vendor/autoload.php';
+
+use Aws\Exception\AwsException;
+use Aws\WorkDocs\WorkDocsClient;
+
+// snippet-end:[workdocs.php.create_folder.import]
+
+
+/**
+ * Create a new folder in your Amazon WorkDocs.
+ *
+ * This code expects that you have AWS credentials set up per:
+ * https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials.html
+ */
+
+// Create a workdocs Client 
+// snippet-start:[workdocs.php.create_folder.main]
+$client = new Aws\WorkDocs\WorkDocsClient([
+    'profile' => 'default',
+    'version' => '2016-05-01',
+    'region' => 'us-east-2'
+]);
+
+$authTokenFilePath = 'token.txt';

```

Comment:
I missed this in my first pass, but for variables only used once, it's more efficient to just declare the values where they're used. You can name them something appropriate to what they are. For example:
```
$file = fopen('path/to/auth/token.txt', 'r');
...
$result = $client->createFolder([
      'AuthenticationToken' => $authToken,
      'Name' => 'folder_name',
      'ParentFolderId' => 'parent_folder_id'
 ]);
```

This idea should be taken generally across all examples whenever a declared variable is only used once.